iroh-net/src/relay/codec.rs

```diff
@@ -101,8 +101,6 @@ pub(crate) enum FrameType {
     ///
     /// Handled on the `[relay::Client]`, but currently never sent on the `[relay::Server]`
     Restarting = 15,
-    /// 32B src pub key + 32B dst pub key + packet bytes
-    ForwardPacket = 16,
```

Comment:
@ramfox this was part of the compilation failures, but I think was part of meshing. Is this correct? it's otherwise not used anywhere